src/Components/Details.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { IoChevronBackOutline } from 'react-icons/io5';
+import Navbar from './Navbar';
+
+const Details = () => {
+  const { id } = useParams();
+  const coins = useSelector((state) => state.coin.coins);
+  const coin = coins.find((coin) => coin.id === id);
+  return (
+    <div>
+      <Navbar title={<IoChevronBackOutline />} />
+      <div className="container-detail">
+        <div className="detail-header d-flex">
+          <img src={coin.icon} alt="coin" />
+          <h2>
+            <span>
+              #
+              {coin.rank}
+            </span>
+            {' '}
+            {coin.name}
+          </h2>
+        </div>
+        <div className="card-detail">
+          <div className="card-body-detail d-flex">
+            <p className="card-title">Name</p>
+            <p className="card-title">{coin.name}</p>
+          </div>
+          <li className="card-body-detail d-flex">
+            <p className="card-title">Symbol</p>
+            <p className="card-title">{coin.symbol}</p>
+          </li>
+          <div className="card-body-detail d-flex">
+            <p className="card-title">Price to USD</p>
+            <p className="card-title">
```

Comment:
- [ ] Kindly follow the given design guidelines for the app.

Given design
![image](https://user-images.githubusercontent.com/81932478/203768852-b4ab5ce5-9e2a-42ab-a71b-9bb4c231e2a1.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/203769054-164d603a-4145-4d4b-8434-ec93fd3c98bc.png)

- The following section below could be displayed like the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/203770409-3ea11981-2dfb-494f-90e0-96f34ea53f0e.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/203770310-ffe74fe5-f6ca-411e-9811-9d875e366254.png)

- Also see the difference between your search bar and the given design
- In the given design, the cards two per row